monitor-iot-data-with-freeboard/scripts/sin-generator-device.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ -n "$1" ]; then
+    DEVICE_ID=$1
+else
+    DEVICE_ID="SIN_GENERATOR"
+fi
+
+PI=3.14159265359
```

Comment:
Use `echo "scale=1000; 4*a(1)" | bc -l` to get more accurate value of PI (1000 digits)
#troll